wpilibc/src/main/native/cpp/Commands/Scheduler.cpp

```diff
@@ -166,10 +166,10 @@ void Scheduler::Run() {
 }
 
 /**
- * Registers a {@link Subsystem} to this {@link Scheduler}, so that the {@link
- * Scheduler} might know if a default {@link Command} needs to be run.
+ * Registers a Subsystem to this Scheduler, so that the Scheduler might know if
```

Comment:
might should be will

Reply:
Why remove the link to Subsystem?  I can understand removing the link to the Scheduler.  Or is the issue that doxygen isn't picking up the links?

Reply:
doxygen automatically turns instances of "Command", et al. into links.